packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -517,11 +520,17 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
     // If the user passed is a value prop, use that
     // unless we are open and have a valid current pending index
-    if (!(isOpen && currentPendingIndexValid) && (text && !currentPendingValue)) {
+    if (
+      !(isOpen && currentPendingIndexValid) &&
+      (text && (currentPendingValue === null || currentPendingValue === undefined))
```

Comment:
>(currentPendingValue === null || currentPendingValue === undefine [](start = 15, length = 65)

Why was this changed? Are we accounting for the case where the currentPendingValue is "" which translates to false?

Reply:
Exactly so if a user is typing a freeform value and they get to an empty string it could (if value or text are present) revert to that value when it should still show an empty string